### DISCUSSION TITLE
Does lego support a configuration file?

### DISCUSSION THREAD

**Discussion by johanneskastl** [go-acme/lego#1594] [Q&A] [answered]
Sorry if this question was asked and answered before, did not find anything:

Does lego support a configuration file, that is automatically read? Something in `/etc/` maybe?
After having a look at the output of `lego --help` it looks like it is meant to be run with only command line arguments?

Thanks in advance!
Johannes

**Comment by ldez** [comment, marked as answer]
Hello,

lego uses flags only, there is no configuration file.

**Reply by johanneskastl** [reply]
Thanks for the quick reply. Would it make sense to open a feature request for this? This would make it easier to run this as a systemd service, without having to handcraft the service file with the exact details one needs.

I am currently packaging lego for openSUSE, hence my questions...
